back-end/sports_platform/sports_platform_api/models/event_models.py

```diff
@@ -65,6 +95,49 @@ def create_event(data):
             return {"@id": event.event_id}
         except Exception as e:
             return 500
+    
+    def _scheme_location(self):
+        return {
+            '@context': 'https://schema.org',
+            '@type': 'Place',
+            'geo': {
+                '@type':'GeoCoordinates',
+                'latitude': float(self.latitude),
+                'longitude': float(self.longitude)
+            },
+            'address': f'{self.district}, {self.city}, {self.country}'
+        }
+    
+    def _scheme_participants(self, participants):
+        return [{"@context":"https://schema.org","@type":"Person","@id": participant.user} for participant in participants]
+
+    def _scheme_additional(self):
+        return [
+            {
+             "@type": "PropertyValue",
+             "name": "minimumAttendeeCapacity",
+             "value": self.minimumAttendeeCapacity
+             },{
+            "@type": "PropertyValue",
+            "name": "maxSpectatorCapacity",
+            "value": self.maxSpectatorCapacity
+             }
```

Comment:
Also interesteds should be added as additionalProperty 
```
{
    "@type": "PropertyValue",
    "name": "interesteds",
    "value": list of users
}
```

Reply:
thanks, it is added